float64 x
float64 y
float64 z
uint8 sys_calib
uint8 gyro_calib
uint8 mag_calib
uint8 accel_calib